polkadot/runtime/common/src/auctions.rs

```diff
@@ -667,1268 +667,8 @@ impl<T: Config> Pallet<T> {
 	}
```

Comment:
can you please move this into a `mod.rs` in the folder?

I think that is more obvious to me.